pkg/cosign/bundle/bundle.go

```diff
@@ -21,6 +21,9 @@ import "github.com/sigstore/rekor/pkg/generated/models"
 type Bundle struct {
 	SignedEntryTimestamp []byte
 	Payload              RekorPayload
+
+	// RFC3161Entry contains the recorded entry from timestamp authority servers.
+	RFC3161Entry []byte
```

Comment:
Will this be the raw bytes, or base64 encoded like the SET?

Reply:
I was thinking of raw bytes.

I haven't find any base64 reference for the fields in the specs 🤔. But I am open to encode/decode these bytes to base64.
 https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto
 https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_common.proto

Reply:
I think @haydentherapper's point was that the `SignedEntryTimestamp []byte` above is actually base64-encoded (I think?). I don't think it *should* be, but putting a new, non-base64-encoded value side-by-side with it in the `Bundle` struct makes it confusing.

Also, when we're dumping this to the OCI registry, do we have to base64-encode at that point? 

Can we just make sure that comments make this all crystal clear?